components/atomic/buttons/ButtonControl/ButtonControl.styles.ts

```diff
@@ -65,7 +63,18 @@ export const ButtonControl = styled.button<BaseProps>`
   }
 `;
 
-export const ButtonControlIcon = styled(ButtonControl)<BaseProps>`
-  padding: ${pxToRem("7px")};
-  min-width: auto;
+export const ButtonText = styled.span<Pick<BaseProps, "size" | "icon">>`
+  display: inline-block;
+
+  ${({ size }) =>
+    size === "large" &&
+    css`
+      padding-inline-start: ${pxToRem("4px")};
+    `}
```

Comment:
In this block, I had to adjust the text's padding for large buttons. Lael's design called for slightly more padding at the start of a button if there's text.